README.md

```diff
@@ -40,8 +61,18 @@ Meanwhile, Please select **Visual Reality Supported** in XR Settings
     <img src="./HoloRepositoryDemoApplication/Images/XRSettings.png" height="200">
 </p>
 
+
 ### Run the App in Game mode
 If you have setup the environment and settings, please open the **BaseScene** in Scenes folder, and press Play Button to try it.
 
 
+## Build and deployment
+> TODO: Please add this section.
```

Comment:
Please fill TODO. Maybe mention emulator as well as how to deploy to the actual hardware. No need to go into detail, you can also just refer to where it is described.